modules/core/src/main/java/org/apache/synapse/config/SynapseConfiguration.java

```diff
@@ -2127,4 +2132,25 @@ private String getTenantDomain(SynapseEnvironment synapseEnvironment) {
         return null;
     }
 
+    /**
+     * Returns CommentedText List
+     *
+     * @return ArrayList of String contains commented text segments
+     */
+    public List<String> getCommentedTextList() {
```

Comment:
Maybe it's a good idea not to expose internals such as the data structures used internally, etc as part of "method names".